cmd/mr_checkout.go

```diff
@@ -24,9 +24,16 @@ var (
 
 // listCmd represents the list command
 var checkoutCmd = &cobra.Command{
-	Use:   "checkout [remote] [<MR id or branch>]",
-	Short: "Checkout an open merge request",
-	Args:  cobra.RangeArgs(1, 2),
+	Use:     "checkout [remote] [<MR id or branch>]",
+	Aliases: []string{"co"},
+	Short:   "Checkout an open merge request",
+	Long: heredoc.Doc(`
+		Checkout an open merge request to using the MR's source branch name as
```

Comment:
I think you can remove "to" and this will still read just fine.

Reply:
Oh, that's probably a left over from a rephrasing :).
Will remove, thanks.